src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -15,37 +15,43 @@ public static class SimplificationOptions
         /// This option tells the simplification engine if the Qualified Name should be replaced by Alias
         /// if the user had initially not used the Alias
         /// </summary>
+        [Obsolete]
```

Comment:
💡 It would be good to add a message here indicating that the option is no longer used.

Reply:
Done!